### PR TITLE
Spelling mistakes

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**

### CODE REVIEW

File: inst/fw_templ/p_example/server_global.R [hunk 1]
```diff
@@ -63,7 +63,7 @@ node_description <- list("global.R"             = "Handles FRAMEWORK global work
                          "img"                  = "Application related images and icons",
                          "loader.gif"           = "buys indicator used image",
```
Comment (agenius-jasmine) [inline inst/fw_templ/p_example/server_global.R:64]: buys -> Busy

File: tests/testthat/sample_app_both_sidebars/program/server_global.R [hunk 2]
```diff
@@ -63,7 +63,7 @@ node_description <- list("global.R"             = "Handles FRAMEWORK global work
                          "img"                  = "Application related images and icons",
                          "loader.gif"           = "buys indicator used image",
```
Comment (agenius-jasmine) [inline tests/testthat/sample_app_both_sidebars/program/server_global.R:64]: buys -> busy

File: tests/testthat/sample_app_left_sidebar/program/server_global.R [hunk 3]
```diff
@@ -63,7 +63,7 @@ node_description <- list("global.R"             = "Handles FRAMEWORK global work
                          "img"                  = "Application related images and icons",
                          "loader.gif"           = "buys indicator used image",
```
Comment (agenius-jasmine) [inline tests/testthat/sample_app_left_sidebar/program/server_global.R:64]: buys -> busy

File: tests/testthat/sample_app_no_both_sidebars/program/server_global.R [hunk 4]
```diff
@@ -63,7 +63,7 @@ node_description <- list("global.R"             = "Handles FRAMEWORK global work
                          "img"                  = "Application related images and icons",
                          "loader.gif"           = "buys indicator used image",
```
Comment (agenius-jasmine) [inline tests/testthat/sample_app_no_both_sidebars/program/server_global.R:64]: buys -> busy

File: tests/testthat/sample_app_right_sidebar/program/fxn/plots.R [hunk 5]
```diff
@@ -12,33 +12,29 @@ attr(mtcars, "show_rownames") <- TRUE
 # -- plotting functions
 
 plot2ggplot <- function() {
-    plot <- ggplot(data = mtcars, aes(x = wt, y = mpg)) +
+    ggplot(data = mtcars, aes(x = wt, y = mpg)) +
         geom_point(aes(color = cyl)) +
         theme(legend.justification = c(1, 1),
               legend.position = c(1, 1),
-              legend.title = element_blank()) +
+              legend.title    = element_blank()) +
```
Comment (agenius-jasmine) [inline tests/testthat/sample_app_right_sidebar/program/fxn/plots.R:19]: do we need to align with legend.justification

**Review by agenius-jasmine — COMMENTED** [review]
minor fixes

**Review by cb4ds — APPROVED** [review]

File: inst/fw_templ/p_example/server_global.R (hunk 1 above)
Comment (agenius-mohammed-ali) [reply inst/fw_templ/p_example/server_global.R:64]: done

File: tests/testthat/sample_app_left_sidebar/program/server_global.R (hunk 3 above)
Comment (agenius-mohammed-ali) [reply tests/testthat/sample_app_left_sidebar/program/server_global.R:64]: done

File: tests/testthat/sample_app_no_both_sidebars/program/server_global.R (hunk 4 above)
Comment (agenius-mohammed-ali) [reply tests/testthat/sample_app_no_both_sidebars/program/server_global.R:64]: done

File: tests/testthat/sample_app_right_sidebar/program/fxn/plots.R (hunk 5 above)
Comment (agenius-mohammed-ali) [reply tests/testthat/sample_app_right_sidebar/program/fxn/plots.R:19]: done

**Pull request merged**